common/lib/dependabot/pull_request_creator/branch_namer.rb

```diff
@@ -29,7 +29,8 @@ def new_branch_name
               elsif dependencies.count > 1 && updating_a_dependency_set?
                 dependency_set.fetch(:group)
               else
-                dependencies.map(&:name).join("-and-").tr(":", "-")
+                dependencies.map(&:name).join("-and-").tr(":", "-").
+                  gsub("@", "at-")
```

Comment:
Moved to the next line because of rubocop 🤷‍♂

Reply:
How about we just sub it out for nothing? I don't think we're likely to see collisions.

```suggestion
                  gsub("@", "")
```